content/en/docs/what-is-grpc/introduction.md

```diff
@@ -103,7 +103,7 @@ a slightly simplified syntax, some useful new features, and supports more
 languages. Proto3 is currently available in Java, C++, Dart, Python,
 Objective-C, C#, a lite-runtime (Android Java), Ruby, and JavaScript from the
 [protocol buffers GitHub repo][], as well as a Go language generator from the
-[golang/protobuf GitHub repo][], with more languages in development. You can
+[golang/protobuf Official package][], with more languages in development. You can
```

Comment:
Nit: `s/O/o/`

Reply:
Totally right, sorry about that, fixed by https://github.com/grpc/grpc.io/commit/1c224c8fa4d383bb866e31cf7bc87cba072a8aba